api/src/main/java/org/openmrs/module/coreapps/parser/ParseEncounterToJson.java

```diff
@@ -68,7 +68,7 @@ public SimpleObject createEncounterJSON(User authenticatedUser, Encounter encoun
         }
         EncounterType encounterType = encounter.getEncounterType();
         simpleEncounter.put("encounterType",
-                SimpleObject.create("uuid", encounterType.getUuid(), "name", uiUtils.format(encounterType)));
+                SimpleObject.create("uuid", encounterType.getUuid(), "name", uiUtils.message(encounterType.getName())));
```

Comment:
Before your changes the entire encounter type representation was passed to the model. Let's try to not break this, could you try this:
```java
EncounterType encounterType = encounter.getEncounterType();
encounterType.setName(uiUtils.message(encounterType.getName()));
simpleEncounter.put("encounterType",
        SimpleObject.create("uuid", encounterType.getUuid(), "name", uiUtils.format(encounterType)));
```

Reply:
Yes ,we can.If you see https://github.com/openmrs/openmrs-module-uiframework/blob/fcc39f0b7e4cccb04834d10d1c5d6e1dec29a523/api/src/main/java/org/openmrs/ui/framework/FormatterImpl.java#L65-L96 , uiutils.format() checks a lot of class and that is overburden ,we already know that at this point we need only encounter type name so instead of using uiutils.format()  ,uiUtils.message() would be better.

Reply:
@mogoodrich do you know if anything else than the encounter type name is used  on those views?

Reply:
Whoops, sorry I didn't catch this earlier, there's already a way to do this, will comment on the ticket.